lib/Dialect/FIRRTL/Analysis/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_circt_dialect_library(CIRCTFIRRTLAnalysis
```

Comment:
I'd just add this to the existing firrtl passes directory.  It isn't really an "analysis" per-say, it is a checker like the CheckWidths pass.